src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,54 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+// Styles
+import './PostInfo.scss';
+
+// Types
+import { Comment } from '../../Types/comment';
+import { User } from '../../Types/user';
+// Components
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  title: string;
+  user: User | null;
+  body: string;
+  comments: Comment[];
+};
+
+// Component
+export const PostInfo: React.FC<Props> = ({
+  title,
+  user,
+  body,
+  comments,
+}) => (
+  <li className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <UserInfo
```

Comment:
You should handle the case when user prop is not provided, by displaying a default message or a loading indicator.

Reply:
isn't fixed